examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1096 @@
23Pq1KliLVBKnj34JCv4iIhI2VKo4NOwYUMWL15MREQEK1eupHv37gDExMTg4+NTrAVKydMaHxERKasKFXzefPNNXn75ZerWrcstt9xCcHAwkN39adWqVbEWKCVPU10iIlJWFepw9oceeojbbruNs2fP2s/hA9ClSxceeOCBYitOzKHgIyIiZVWhgg9AQEAAAQEBnDlzBovFQs2aNXXywjKiilf2VFdMUgyGYeS6iKmIiIgjK9RUl81mY+zYsfj6+lKnTh1q166Nn58fb731FjabrbhrlBLm7+kPQIYtg8T0RJOrERERKT6F6vi8/vrrzJo1i/fee48OHTpgGAYbN25k9OjRpKam8s477xR3nVKCPF088XTxJDkjmdjkWLzdvM0uSUREpFgUKvjMnTuXmTNn2q/KDtCyZUtq1qzJc889p+BTBvh7+nM67jSxybHUq1jP7HJERESKRaGmui5evEjjxo3zbG/cuDEXL14sclFivpzprtjkWJMrERERKT6FCj4tW7ZkypQpebZPmTKFFi1aFLkoMZ+Cj4iIlEWFmur64IMPuPvuu1mzZg3BwcFYLBY2bdpEREQEy5cvL+4axQQKPiIiUhYVquNz++23c+TIER544AEuX77MxYsX6dWrF/v372f27NnFXaOYwN9DwUdERMqeQp/Hp0aNGnkWMe/Zs4e5c+fyxRdfFLkwMZc6PiIiUhYVquMjZZ89+KQo+IiISNnhEMHn5MmTDBw4kHr16uHh4UGDBg0YNWoU6enpecbOmTOHFi1a4O7uTkBAAEOHDjWhYsenjo+IiJRFhZ7qKkmHDh3CZrMxffp0GjZsSFhYGE8//TRJSUmMHz/ePm7ixIlMmDCBDz/8kHbt2pGamsrx48dNrNxxKfiIiEhZVKDg06tXr2s+fvny5SIVczWhoaGEhoba79evX5/Dhw8zbdo0e/C5dOkSI0eOZOnSpXTp0sU+tmnTptelprJOwUdERMqiAgUfX1/ff3z88ccfL1JB+RUXF0elSpXs91evXo3NZiMyMpKgoCASEhK49dZbmTBhAoGBgVfdT1paGmlpafb78fHx17VuR5ETfC4kX8Bm2LBaHGJWVERE5JoKFHxKy6Hqx44dY/LkyUyYMMG+7fjx49hsNt59910mTZqEr68vI0eOpFu3buzduxdXV9cr7mvcuHGMGTOmpEp3GJU9KwOQZWSx6+wuqlWoRi2fWiZXJSIiUjSm/m/86NGjsVgs17xt374913OioqIIDQ2ld+/eDBo0yL7dZrORkZHBJ598Qo8ePWjfvj0LFy7k6NGj/Pbbb1etYfjw4cTFxdlvERER1+39OhJXJ1d83HwAaPN5Gzp80QGbYTO5KhERkaIxdXHz0KFD6dOnzzXH1K1b1/5zVFQUISEhBAcHM2PGjFzjqlevDkCTJk3s26pUqYK/vz+nT5++6v7d3Nxwc3MrRPVln7+nP/Fp2VN/p+NOExkfSaDv1acNRURESjtTg4+/vz/+/v75GhsZGUlISAitW7dm9uzZWK25m1UdOnQA4PDhw9SqlT0lc/HiRWJjY6lTp07xFl5OVPaozPFL/zsq7siFIwo+IiLi0BxixWpUVBSdO3cmMDCQ8ePHc/78eaKjo4mOjraPadSoET179uSFF15g06ZNhIWF8cQTT9C4cWNCQkJMrN5xnYk/k+v+kQtHTKpERESkeDjEeXxWrVpFeHg44eHh9m5ODsMw7D9/+eWXvPjii9x9991YrVZuv/12VqxYgYuLS0mXXCZUcK2Q676Cj4iIODqL8dfkIMTHx+Pr60tcXBw+Pj5ml2Oqjac3Mur3UTSv2pyPt37MXTfcxc+P/mx2WSIiInnk9/vbIaa6xBwdandgzeNr6Nm4J6COj4iIOD4FH/lHjSo3AuDEpROkZ+W9PpqIiIijUPCRf1S9QnW8XLzIMrI4cemE2eWIiIgUmoKP/COLxWLv+mi6S0REHJmCj+RLTvAZ+8dYtpzZYnI1IiIihaPgI/nSt1lfXJ1c2R61ne7zuuvyFSIi4pAUfCRfejbuyaEhhwBISE8gJSPF5IpEREQKTsFH8q2uX10sWIDs8CMiIuJoFHwk3ywWi/1szonpiSZXIyIiUnAKPlIgCj4iIuLIFHykQBR8RETEkSn4SIyDDPMAACAASURBVIEo+IiIiCNT8JECUfARERFHpuAjBaLgIyIijkzBRwpEwUdERByZgo8UiLerN6DgIyIijknBRwpEHR8REXFkCj5SIAo+IiLiyBR8pEAUfERExJEp+EiBKPiIiIgjU/CRAskJPrpIqYiIOCIFHykQdXxERMSRKfhIgSj4iIiII1PwkQJR8BEREUem4CMFouAjIiKOTMFHCkTBR0REHJmCjxTIX4OPYRgmVyMiIlIwCj5SIN5u2dfqshk2UjNTTa5GRESkYBR8pEA8XTztP2u6S0REHI2CjxSI1WLFy8ULUPARERHHo+AjBaYFziIi4qgUfKTAFHxERMRRKfhIgSn4iIiIo1LwkQLThUpFRMRRKfhIganjIyIijkrBRwpMwUdERByVgo8UmIKPiIg4KgUfKTAFHxERcVQKPlJgCj4iIuKoFHykwLxds6/XpeAjIiKORsFHCkwdHxERcVQKPlJgCj4iIuKoFHykwBR8RETEUSn4SIEp+IiIiKNS8JECU/ARERFHpeAjBabgIyIijkrBRwpMFykVERFHpeAjBfbXjo9hGCZXIyIikn8KPlJgOcEn05ZJela6ydWIiIjkn4KPFJiXq5f9Z63zERERR6LgIwXmbHXG3dkdUPARERHHouAjhaLrdYmIiCNS8JFC0SHtIiLiiBR8pFAUfERExBEp+EihKPiIiIgjUvCRQlHwERERR6TgI4Wi4CMiIo5IwUcKRcFHREQckYKPFIqCj4iIOCIFHykUXahUREQckYKPFIo6PiIi4ogUfKRQFHxERMQRKfhIoeiSFSIi4ogUfKRQ1PERERFHpOAjhaLgIyIijsghgs/JkycZ
```

Comment:
For now I would suggest to drop this section until we have looked at running Lighting on a bigger box - the statements are too general. We should not recommend SV1 if, say, Lighting on a bigger box would be better. How far can we push this example?